.cirrus.yml

```diff
@@ -1,13 +1,57 @@
-task:
-  name: Build with AOSP
-  only_if: $CIRRUS_PR_LABELS =~ ".*aosp-build.*"
-  timeout_in: 70m
-  container:
-    image: ubuntu:23.04
-    cpu: 8
-    memory: 32G
-  build_script:
-    - ./.github/scripts/build_aosp.sh aosp_arm64 ap1a userdebug android-14.0.0_r29
+container:
+  image: ghcr.io/cirruslabs/android-sdk:34
```

Comment:
We'll need to wait until they have an image for Android 15: https://github.com/cirruslabs/docker-images-android/pkgs/container/android-sdk

Reply:
Or just try setting it to `android-35` below...